lib/wallaby/query.ex

```diff
@@ -289,4 +299,8 @@ defmodule Wallaby.Query do
         |> Keyword.put_new(:maximum, opts[:maximum])
     end
   end
+
+  defp add_get(opts) do
+    Keyword.put_new(opts, :get, nil)
```

Comment:
I'd like to avoid using `nil` here. What do you think about defaulting this to something like `:all`. That way its more descriptive. We can pattern match on that in `result/1`